shared/auth.py

```diff
@@ -806,9 +806,20 @@ async def interact(self):
         except BaseException as exc:
             return await self.failure("PSBT output failed", exc)
 
-        from glob import NFC
+        from glob import NFC, settings
 
         if self.do_finalize and txid and not hsm_active:
+
+            # if NFC PushTx is enabled, do that w/o questions.
```

Comment:
meh for starting without confirmation of action. Would be much better inside the loop on line 826. This way users can export multiple times on possible failure

Reply:
disagree. they enabled it so they want it. already loops for multiple attempts when the tap fails / phone craps out, etc.